README.md

```diff
@@ -401,19 +401,6 @@ instead of installing LocalStack on the current machine. The only dependency is
 installed locally. The test runner will automatically pull the image and start the container for the
 duration of the test.  The container can be configured by using the @LocalstackDockerProperties annotation.
 
-```
-@RunWith(LocalstackDockerTestRunner.class)
-@LocalstackDockerProperties(services = { "sqs", "kinesis:77077" })
```

Comment:
Thanks for this fix @edwinwilson . In fact, `LocalstackDockerTestRunner` is the default in the latest version (has been renamed from `LocalstackDockerTestRunner` to `LocalstackTestRunner`).

I'll merge the PR, but will follow up with a few more required updates in the docs. Thanks